content/en/synthetics/guide/browser-tests-totp.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Time-based one time passwords (TOTPs) for multi-factor authentication (MFA)
+kind: guide
+further_reading:
+   - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+     tag: 'Blog'
+     text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+   - link: 'synthetics/settings/?tab=specifyvalue#global-variables'
+     tag: 'Documentation'
+     text: 'Learn more about global variables'
+   - link: 'synthetics/browser_tests/actions/'
+     tag: 'Documentation'
+     text: 'Learn more about browser test steps'
+ 
+---
+
+### Store your secret key or QR code in a global variable
+
+In the **Global Variables** tab of your **Settings** page, click **Create Global Variable** and enter a secret key or upload a QR code from your authentication provider:
+1. In **Choose variable type**, select **MFA Token**.
+2. In **Define variable**, enter a **Variable Name**. Your variable name can only use uppercase letters, numbers, and underscores.
+3. Enter a **Description** for your variable (optional).
+4. Select **Tags** to associate with your variable (optional).
+5. Enter the **Secret Key** to your variable or upload a QR code image.
+6. Click **+ Generate** to create a TOTP. You can copy the generated TOTP with the **Copy** icon.
+7. In **Permissions settings**, restrict access to your variable based on roles in your org. For more information about roles, see the [RBAC documentation][1].
```

Comment:
7. is not available to all users but only the ones with the custom roles flag. Let's clarify that in a way that's not too different from what we have on https://docs.datadoghq.com/synthetics/settings/?tab=createfromhttptest#restrict-access.